content/en/docs/concepts/security/index.md

```diff
@@ -673,12 +743,13 @@ metadata:
   name: allow-all
   namespace: default
 spec:
+  action: ALLOW
   rules:
   - {}
 {{< /text >}}
 
-The example below shows a simple policy `deny-all` which denies access to all workloads
-in the `admin` namespace.
+The example following example shows a simple  `deny-all` policy that denies any
```

Comment:
Given we have "DENY" semantic now, should we rewrite this sentence to avoid confusion. E.g "The example following example shows a policy that does not allow any access to all workloads in the `admin` namespace."

May also show the recommendation and example to use DENY policy for such use case

+ @yangminzhu

Reply:
@diemtvu Thanks for pointing it out. I also fixed the duplicated example.

Reply:
Yes, I think that makes sense:

We can change the current one to:
```
The following example shows a policy that does not allow any access to all workloads in the admin namespace. You can add more allow policies to allow extra access later.

{{< text yaml >}}
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: allow-nothing
  namespace: admin
spec:
  action: ALLOW
  rules:
  - {}
{{< /text >}}

The following example shows a policy that explicitly denies any access to all workloads in the `database` namespace. You can't add allow policies to allow extra access because the deny policy takes precedence over allow policy, meaning it will deny the request before any allow policies.

{{< text yaml >}}
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
  name: deny-all
  namespace: database
spec:
  action: DENY
  rules:
  - {}
{{< /text >}}

Reply:
@rcaballeromx I just made the comment, could you update with the latest change? thanks.

Reply:
@yangminzhu The first example yaml  has an `ALLOW` action. How can a reader know that it is denying access?

Reply:
I just fixed a minor issue in my comment. The `allow-nothing` doesn't have `rules` which means it will never match, which means it allows nothing and any request will be denied.

Reply:
@yangminzhu Thanks for the correction. It is still confusing that readers need to use an allow policy to deny any requests. It is also not clear what the difference between `allow-nothing` and `deny-all` is. I can see how they both could be used depending on preference, but do we really want to introduce folks to both on the concept page? I would rather only keep the `deny-all` policy and keep things simple. @justinpettit WDYT?

Reply:
I agree that we're adding complexity to a document that should probably be higher-level.  @yangminzhu, can we make a note of this and find a more appropriate location post-1.5?

Reply:
@rcaballeromx the difference between `allow-nothing` and `deny-all` is the interaction between other policies. 

- If you have `allow-nothing`, it first restricts the permission to zero and you can incrementally add more allow policies to grant more permission. 

- If you have `deny-all`, all requests will be denied even if you add other allow policies, you need to delete the `deny-all` policy to allow other access.

We can keep the `deny-all` but please replace the current example with the one from my comment. That's a better way to express `deny-all` semantics.

Reply:
@rcaballeromx This comment is not addressed in the merged PR, Rigs, would you mind to send out a follow PR to fix this? Thanks.